src/iOS/AppDelegate.cs

```diff
@@ -168,6 +168,11 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
                         await ASCredentialIdentityStore.SharedStore?.RemoveAllCredentialIdentitiesAsync();
                     }
                 }
+                else if ((message.Command == "softDeletedCipher" || message.Command == "restoredCipher")
```

Comment:
Shouldn't `softDeletedCipher` message just be handled like `deletedCipher` and `restoredCipher` be treated like `addedCipher`?